bands/templates/bands/index.html

```diff
@@ -69,7 +69,7 @@
     <tr>
         <td colspan="3">
             {% if band.info.completed %}
-                You have already provided all the information required as part of your badge agreement.  You may edit your
+                You have already provided all the information required as part of your band agreement.  You may edit your
```

Comment:
Oh, good catch.  I kind of assumed it was "badge agreement" but "band agreement" makes a lot more sense.